src/index.js

```diff
@@ -153,7 +153,7 @@ export function app(state, actions, view, container) {
         typeof value === "function" ||
         (name in element && name !== "list" && !isSVG)
       ) {
-        element[name] = value == null ? "" : value
+        element[name] = value == null || value === "false" ? "" : value
```

Comment:
or we can use regex here: `/^false$/i.test(value)`
it will be more robust (users can use `False`, `FALSE` etc) but slower
```js
// as it
<input spellcheck="false" /> // => <input spellcheck="false" />
<input spellcheck="False" /> // => <input spellcheck="true" />
<input spellcheck="FALSE" /> // => <input spellcheck="true" />

// regexp
<input spellcheck="false" /> // => <input spellcheck="false" />
<input spellcheck="False" /> // => <input spellcheck="false" />
<input spellcheck="FALSE" /> // => <input spellcheck="false" />
```

Reply:
HTML spec allow different cases ?

Reply:
Yes, HTML case insensitive